### DISCUSSION TITLE
I want to add new feature, called temporary highlight pen

### DISCUSSION THREAD

**Discussion by Belalazam** [xournalpp/xournalpp#5707] [Ideas]
Hi, I want to make contribution in open source project, so i picked this xournal++,  i want to add a pen which will throw ink for temporary
and then vanish after user selected seconrds , this will help tutor to mark the things and no need to delete it , since it will vanish by itself
please give me green flag on to this, so that i can start
thanks

**Comment by rolandlo** [comment]
A "laser" tool for temporary highlighting is very much on the wish list of our users. See #1778, #1822 and #3043 for some discussions around that. So if you come up with a nice implementation we would certainly be interested.

**Comment by Belalazam** [comment]
Thanks, I will try my best

Regards, belal azam

**Comment by Mel-2108** [comment]
I've had the same idea and as it seems, many other people do since 2020. It might be useful to have some sort of a plugin development chat so we can collaborate and create plugins faster together. I don't know if something like that already exists though.
